exercises/react/.meta/plugins.py

```diff
@@ -0,0 +1,20 @@
+def add_callbacks(case):
+    observers = []
+    callbacks = []
+    operations = list(filter(lambda op: op["type"] == "add_callback", case["input"]["operations"]))
+
+    if len(operations) == 0:
+        return '', {}
+
+    sufix = ''
```

Comment:
I think this should probably be `suffix`?

Reply:
yep :)

Reply:
it is prefix actually